xcell/cls/cl.py

```diff
@@ -711,17 +778,18 @@ def get_cl_file(self):
             fname = os.path.join(self.outdir, f'cl_{self.tr1}_{self.tr2}.npz')
         ell = np.arange(3 * nside)
         if not os.path.isfile(fname):
+            print(f"Computing fiducial Cell for {self.tr1} {self.tr2}")
             ccl_tr1, ccl_tr2 = self.get_tracers_ccl()
             cl = self.th.get_ccl_cl(ccl_tr1, ccl_tr2, ell)
             b1 = self.data.get_bias(self.tr1)
             b2 = self.data.get_bias(self.tr2)
             cl *= b1*b2
             tracers = self.data.data['tracers']
-            fiducial = self.data.data['cov']['fiducial']
             d1, d2 = self.get_dtypes()
-            for dtype, tr in zip([self.tr1, self.tr2], [d1, d2]):
-                if (dtype == 'galaxy_shear') and fiducial['wl_m']:
-                    cl = (1 + tracers[tr]['m']) * cl
+            for tr, dtype in zip([self.tr1, self.tr2], [d1, d2]):
+                if (dtype == 'galaxy_shear'):
+                    m = tracers[tr].get('m', 0)
+                    cl = (1 + m) * cl
```

Comment:
humm, why are we doing this? Is this only for the covariance? Or will you end up storing C_ells in the sacc file that are already corrected for the multiplicative bias?

Reply:
This will store fiducial Cells that are corrected for the multiplicative bias. I'm doing this to do the marginalization of `m` correctly.

Reply:
so only for the fiducials, not for the final sacc file, right?

Reply:
Yep. It's inside the `ClFid` class.